docs/kafka/topic-configuration-kafka/README.adoc

```diff
@@ -535,14 +582,17 @@ Log segment size is the size of the log segment files that constitute the log. L
 [cols="25%,75%"]
 |===
 h|Editable
-|No
+|Yes
 
 h|Type
 |Integer
 
 h|Default value
 |`1 GB`
 
+h|Supported values
+| 50 MB or greater
```

Comment:
Because the unit is `byte`, it should change to `52428800` instead.
ex: 52428800 (50 MB) or greater

WDYT?

Reply:
And the default value `1 GB` is not correct. It should be `1073741824` bytes = 1GB